.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@
 
 ## Why It's Good For The Game
 
-<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
+<!-- Argue why these changes would benefit the game to the community, especially if they are controversial and/or far reaching. If you can't justify it in words, it might not be worth adding. -->
```

Comment:
applies here too

Reply:
changed to:
`Argue for the merits of your changes and how they benefit the game, especially if they are controversial and/or far reaching. If you can't actually explain WHY what you are doing will improve the game, then it probably isn't good for the game in the first place.`